common/common-client/src/main/java/org/ow2/proactive/authentication/jaas.config

```diff
@@ -12,6 +12,32 @@ SchedulerPAMLoginMethod {
 	org.ow2.proactive.scheduler.authentication.SchedulerPAMLoginModule required serviceName="proactive-jpam";
 };
 
+SchedulerIAMLoginMethod {
+    org.ow2.proactive.authentication.IAMLoginModule required
+	ticketValidatorClass="org.ow2.proactive.authentication.iam.JWTValidator"
+	iamServerUrlPrefix="https://localhost:8444/iam"
+	iamTicketRequest="/v1/tickets"
+	service="PACoreWithJWT"
+	userAttributeName="uid"
+	roleAttributeName="role"
+	roleSeparator=","
+	ssoTicketHeader="Location"
+	jsonWebTokenSigned="false"
+	jsonWebTokenEncrypted="false"
+	jsonWebTokenSignatureKey=""
+	jsonWebTokenEncryptionKey="";
+};
+
+/*SchedulerIAMLoginMethod {
```

Comment:
Do you still need this commented out configuration?

Reply:
Yes, this commented configuration is needed when we use a different TicketValidator class. I prefer keeping it to help users configure JAAS easily.

Reply:
Ok good point.